x-pack/plugins/task_manager/server/metrics/lib/simple_histogram.ts

```diff
@@ -23,6 +23,7 @@ export class SimpleHistogram {
   private maxValue: number;
   private bucketSize: number;
   private histogramBuckets: Bucket[] = [];
+  private allValues: number[] = [];
```

Comment:
To track the actual values, we just changed the SimpleHistogram class we use to create histogram shapes, to also track each value passed in.  And add `getAllValues()` to return it.